libdieharder/wyhash.h

```diff
@@ -0,0 +1,140 @@
+//Author: Wang Yi <godspeed_china@yeah.net>
```

Comment:
Misses a license statement. Ability to distribute disputable!

Reply:
I added thre source in the commit: See https://github.com/wangyi-fudan/wyhash/blob/master/LICENSE